source/common/network/utility.cc

```diff
@@ -355,8 +355,19 @@ Address::InstanceConstSharedPtr Utility::getOriginalDst(Socket& sock) {
         sock.getSocketOption(SOL_IPV6, IP6T_SO_ORIGINAL_DST, &orig_addr, &addr_len).return_value_;
   }
 
+#ifdef IP_TRANSPARENT
```

Comment:
seems like you should check ip transparent available or not by https://github.com/envoyproxy/envoy/blob/768d0bdc25c19e0088e1af105b7d65507e47d1b9/source/common/api/posix/os_sys_calls_impl.h#L31, not only the macro

Reply:
have fixed this check, could you pls take a look again, thanks